docs/source/io_formats/index.rst

```diff
@@ -1,13 +1,15 @@
-.. _usersguide_output:
+.. _io_file_formats:
 
-===================
-Output File Formats
-===================
+===============
+IO File Formats
+===============
 
 .. toctree::
    :numbered:
    :maxdepth: 3
 
+   data_wmp
+   mgxs_library
```

Comment:
Hint, hint, nudge, nudge? 😉
On May 26, 2016 9:51 AM, "Paul Romano" <notifications@github.com> wrote:

> In docs/source/io_formats/index.rst
> <https://github.com/mit-crpg/openmc/pull/618#discussion_r64748299>:
>
> >
> >  .. toctree::
> >     :numbered:
> >     :maxdepth: 3
> >
> > +   data_wmp
> > +   mgxs_library
>
> It's a little weird to me to have an XML format here alongside a bunch of
> binary formats.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/618/files/5a9dc631357d5bdd7360b1bd22a1fe8f1576bd82#r64748299>
>